wpmautic.php

```diff
@@ -141,21 +141,55 @@ function wpmautic_base_script() {
  * @return void
  */
 function wpmautic_inject_script() {
-	$base_url = wpmautic_base_script();
```

Comment:
Splitting this function in multiple ones seems very complicated. What do you think to move the `wpmautic_inject_base_script` and `wpmautic_inject_send_script` logic directly in the `wpmautic_inject_script` function ?

Reply:
I generally prefer small functions with a clear purpose. I think that helps to read the code. In my experience it also helps if you like to overwrite certain functionality of the code. But your code base your rules :) Feel free to change it.